### ISSUE TITLE
not listed in community plugins

### ISSUE THREAD

**Issue by ksdavidc** [martinniee/Obsidian-fast-image-cleaner#18]
is not/no longer listed in the community plugins. Only installable via BRAT

**Comment by martinniee** [comment]
Yes, you are correct. Since the recent update, I haven't continued developing it for quite some time. This is because after setting the GitHub repository to archived status, I noticed that the plugin was no longer visible in the community plugin list. I believe this might be due to a rule or mechanism regarding how the Obsidian plugin community list retrieves and displays plugins from GitHub. Fortunately, another developer has taken over this plugin, forked it, and added additional features related to image attachments, making it even more powerful. Therefore, this repository will no longer be updated. You may want to try the improved plugin (attachflow).&nbsp; Thank for your support and like this plugin.
